tests/subsys/ipc/ipc_sessions/testcase.yaml

```diff
@@ -8,28 +8,46 @@ common:
   harness: ztest
 
 tests:
-  sample.ipc.ipc_sessions.nrf5340dk:
+  ipc.ipc_sessions.nrf5340:
     platform_allow:
       - nrf5340dk/nrf5340/cpuapp
+      - nrf5340bsim/nrf5340/cpuapp
     integration_platforms:
       - nrf5340dk/nrf5340/cpuapp
-  sample.ipc.ipc_sessions.nrf54h20dk_cpuapp_cpurad:
+      - nrf5340bsim/nrf5340/cpuapp
+  ipc.ipc_sessions.nrf5340bsim_cpuapp_no_unbound_cpunet:
```

Comment:
These should also work with hardware, but as I have not tested that I only added `nrf5340bsim` to the integration platforms